core/domain/fs_domain.py

```diff
@@ -17,10 +17,10 @@
 """Domain objects representing a file system and a file stream."""
 
 from __future__ import annotations
-
 from core import feconf
 from core import utils
 from core.platform import models
+from typing import Any, List
```

Comment:
Can you please also add typing to fs_domain_test.py and remove both from https://github.com/oppia/oppia/blob/develop/scripts/run_mypy_checks.py#L40

Reply:
done